homeassistant/components/jvc_projector/strings.json

```diff
@@ -37,6 +37,18 @@
       "jvc_power": {
         "name": "[%key:component::sensor::entity_component::power::name%]"
       }
+    },
+    "sensor": {
+      "jvc_power_status": {
+        "name": "Power status",
+        "state": {
+          "standby": "[%key:common::state::standby%]"
```

Comment:
Oops looks like you/I missed a `,`

Reply:
Be sure to run `python3 -m script.translations develop` locally and check the values render ok in the UI as well.